apps/gallery/index.html

```diff
@@ -172,7 +172,7 @@ <h1 id="crop-header" data-l10n-id="cropimage"></h1>
 
     <section role="region" id="edit-view" class="skin-dark">
       <gaia-header ignore-dir id="edit-header" action="close" title-start="50" no-font-fit>
-        <h1 id="edit-title" data-l10n-id="edit"></h1>
+        <h1 id="edit-title" data-l10n-id="edit" hidden></h1>
         <button id="edit-save-button" data-l10n-id="save">Save</button>
         <button id="edit-tool-apply-button" data-l10n-id="apply" hidden>Apply</button>
```

Comment:
Hiding the h1 element without hiding the button elements still leaves the header visible, doesn't it?   It seems like it would be cleaner to hide the entire header if you can do that easily.  Or, since the bug is only for the pick header, and that one doesn't have buttons, maybe you could just do the hidden attribute in that case.